delivery.yaml

```diff
@@ -1,15 +1,23 @@
 version: "2017-09-20"
 pipeline:
 - id: build
+  overlay: ci/golang
+  cache:
+    paths:
+    - /go/pkg/mod       # pkg cache for Go modules
+    - ~/.cache/go-build # Go build cache
   type: script
   commands:
-  - desc: Build and push docker image
+  - desc: Build docker image
```

Comment:
We should also run `make test` before building via CDP.

Reply:
I wasn’t sure about that because we already run make test in travis.

Reply:
I would say we should have it to ensure we only build and push an image if the tests pass.